showcase/app/components/shw/divider/index.gts

```diff
@@ -14,7 +14,10 @@ export interface ShwDividerSignature {
 }
 
 const ShwDivider: TemplateOnlyComponent<ShwDividerSignature> = <template>
-  <hr class="shw-divider {{if @level (concat 'shw-divider--level-' @level)}}" />
+  <hr
+    class="shw-divider {{if @level (concat 'shw-divider--level-' @level)}}"
+    ...attributes
```

Comment:
I needed to add extra space above the divider line for my Showcase examples using tooltips.